cmd/cap1188/main.go

```diff
@@ -0,0 +1,157 @@
+// Copyright 2017 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
```

Comment:
Be sure to include an empty line before the package name otherwise the copyright will be considered package documentation. :)